README.rst

```diff
@@ -317,6 +317,144 @@ specified in ``$HOME/.teuthology.yaml``::
 
     test_path: <directory>
 
+OpenStack backend
+=================
+
+The ``teuthology-openstack`` command is a wrapper around
+``teuthology-suite`` that transparently creates the teuthology cluster
+using OpenStack virtual machines.
+
+Prerequisites
+-------------
+
+An OpenStack tenant with access to the nova and cinder API (for
+instance http://entercloudsuite.com/). If the cinder API is not
+available (for instance https://www.ovh.com/fr/cloud/), some jobs
+won't run because they expect volumes attached to each instance.
+
+Setup OpenStack at Enter Cloud Suite
+------------------------------------
+
+* create an account and `login the dashboard <https://dashboard.entercloudsuite.com/>`_
+* `create an Ubuntu 14.04 instance
+  <https://dashboard.entercloudsuite.com/console/index#/launch-instance>`_
+  with 1GB RAM and a public IP and destroy it immediately afterwards.
+* get $HOME/openrc.sh from `the horizon dashboard <https://horizon.entercloudsuite.com/project/access_and_security/?tab=access_security_tabs__api_access_tab>`_
+
+The creation/destruction of an instance via the dashboard is the
+shortest path to create the network, subnet and router that would
+otherwise need to be created via the neutron API.
+
+Setup OpenStack at OVH
+----------------------
+
+It is cheaper than EnterCloudSuite but does not provide volumes (as
+of August 2015) and is therefore unfit to run teuthology tests that
+require disks attached to the instance. Each instance has a public IP
+by default.
+
+* `create an account <https://www.ovh.com/fr/support/new_nic.xml>`_
+* get $HOME/openrc.sh from `the horizon dashboard <https://horizon.cloud.ovh.net/project/access_and_security/?tab=access_security_tabs__api_access_tab>`_
+
+Setup
```

Comment:
All the steps happen on your host. Only integration testing in the chapter below happen on the vm that was created. This is indeed confusing and I'm assuming the person willing to run the integration tests will have enough motivation to sort that out. 

Any idea how to clarify that there is no need to ssh to something for the steps below ?